### PR TITLE
Styles

### PR DESCRIPTION
**Pull request by Meltrust**
### In this milestone I:

- Implemented basic required styles.
- Implemented basic validation.
- Used localStorage, and implemented DOM event listeners.
- Used git-flow (main << development << feature).
- Delivered the project with an organized folder structure.
- Wrote a descriptive readme file.
- Used the simplest solutions as possible, over the most complex ones.  Readability above all was always the goal.

Thanks for the review! :smile:

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -4,31 +4,38 @@
...
+            class="book-list m-auto border border-dark ps-0 my-4"
+          ></ul>
+        </div>
+
+        <div id="formDiv" class="my-4">
+          <form action="submit" id="main-form" class="m-auto add-form">
+            <h2 class="text-center my-4">Add a New Book</h2>
+            <input type="text" name="Title" placeholder="Title" id="title" />
+            <input type="text" name="Author" placeholder="Author" id="author" />
+            <input type="submit" value="submit" class="btn btn-secondary" />
```
Comment (hemantso) [inline index.html]: - [x] There should be some margin between the input field and submit button as shown in the wireframe. Kindly try to fix it.
Your
image

Original  
image

**Review by hemantso — CHANGES_REQUESTED** [review]
Hi @Meltrust , 

Good job so far! 👍 
There are some issues that you still need to work on to go to the next project but you are almost there! 

Alt Text
## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- It will be good if we will add a live link in the README file.

Cheers and happy coding 😃

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. You can mention me (@hemant-soni-vst-au4) so I can see your comments and respond.


**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (rloterh) [reply index.html]: This issue is well resolved! ✔️

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Miguel, 

Your project is complete! Great job for making the necessary changes recommended by the previous reviewer. There is nothing else to say other than... it's time to merge it 🍾 🍾 :shipit: .
Congratulations! 🎉 🎉 🥂


<br>
Cheers and Happy coding!👏 💻 🍷 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 1 more limited review per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**